src/main/webapp/resources/js/pages/projects/linelist/components/Table/Table.jsx

```diff
@@ -213,46 +217,60 @@ export class TableComponent extends React.Component {
     return `${date}-${project}-${template}.${ext}`;
   };
 
-  createFile = ext => {
+  createFile = (ext) => {
     /*
      * Lazy load xlsx utilities since exporting is not a function used on every page.
      */
     import(
       /* webpackChunkName: "exportUtilities" */ "../../../../../utilities/export-utilities"
-    ).then(module => {
+    ).then((module) => {
       const createXLSX = module.default;
-
-      const availableNames = {};
-      this.props.fields.forEach(f => (availableNames[f.field] = f.headerName));
-
-      /*
-       * Get the visible columns.  Need to ignore the icon columns since
-       * it does not contain any data that we want.
-       */
-      const colOrder = this.columnApi
-        .getColumnState()
-        .filter(c => !c.hide && c.colId !== "icons");
-
-      const data = [];
-      this.api.forEachNodeAfterFilter(node => {
-        // We only need the value of the cell.
-        const item = colOrder.map(col => node.data[col.colId] || "");
-        data.push(item);
+      notification.open({
+        key: "export-notification",
+        message: "Preparing file for download",
```

Comment:
Localize this message and the description

Reply:
Updated in 2e6cbc5d33416e415106f0e72785a23ef36db19f